### PR TITLE
BaSP-MR-111: Activities redux get delete logic

### PR DESCRIPTION
**Pull request by lorenzomauro59**
- Replace the data from the local state.
- Move the logic of the delete to the thunk file in the activities folder and apply it to the entity index file.
- To make the user experience more pleasant, for each request, a spinner (isLoading) should be implemented so that the user understands that a request is being made, for example when loading the table.

### CODE REVIEW

**Comment by lorenzomauro59** [comment]
[[Activities] - Move get and delete logic.](https://trello.com/c/8eW8SdBi/111-activities-move-get-and-delete-logic)

File: src/Redux/activities/thunks.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {
+  getActivitiesPending,
+  getActivitiesSuccess,
+  getActivitiesError,
+  deleteActivitiesPending,
+  deleteActivitiesSuccess,
+  deleteActivitiesError
+} from './actions';
+
+export const getActivities = () => {
```
Comment (EmaBaldor) [inline src/Redux/activities/thunks.js:13]: Should also be getById.

**Review by EmaBaldor — APPROVED** [review]

**Review by EmaBaldor — COMMENTED** [review]
Good job!

**Review by Gonzarey96 — APPROVED** [review]
I checked the code thoroughly for ES5 usage that could be replaced by ES6, inconsistencies, inconsistencies, comments, console.log and ways to optimize this code but I could not find any, congratulations!

**Comment by araefedg** [comment]
The applied functionality works correctly with respect to what was expected,  very good!

**Pull request merged**